runtime/altair/src/migrations.rs

```diff
@@ -10,231 +10,4 @@
-			);
-
-			pre_state.iter().zip(queued.into_iter()).for_each(
-				|((pre_acc, pre_session_key), (post_acc, post_session_key))| {
-					assert!(pre_acc == &post_acc, "Failed to match old validator key");
-					assert!(
-						&pre_session_key.aura == &post_session_key.aura,
-						"Failed to match old aura key"
-					);
-					assert!(
-						&pre_session_key.aura == &post_session_key.block_rewards,
-						"Block rewards key does not match aura one"
-					);
-				},
-			);
-
-			Ok(())
-		}
-	}
-}
-
-mod asset_registry {
-	use super::*;
-
-	/// This migration sets the AssetMetadata.location of all the Tranche tokens
-	/// registered in the AssetRegistry to `None`.
-	pub struct TrancheLocationMigration;
-
-	impl OnRuntimeUpgrade for TrancheLocationMigration {
-		fn on_runtime_upgrade() -> Weight {
-			for (asset_id, metadata) in orml_asset_registry::Metadata::<Runtime>::iter() {
-				if matches!(asset_id, CurrencyId::Tranche(_, _)) && metadata.location.is_some() {
-					match OrmlAssetRegistry::do_update_asset(
-						asset_id,
-						// decimals
-						None,
-						// name
-						None,
-						// symbol
-						None,
-						// existential_deposit
-						None,
-						// location: we do set it to `None`
-						Some(None),
-						// additional
-						None,
-					) {
-						Err(e) => log::error!("TrancheLocationMigration: Failed to update asset with underlying error: {:?}", e),
-						_ => continue,
-					}
-				}
-			}
-
-			// todo(nuno): not sure how to build this properly,
-			// setting it to a conservative value for now.
-			Weight::from_ref_time(200_000_000)
-		}
-
-		#[cfg(feature = "try-runtime")]
-		fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>, &'static str> {
-			Ok(Default::default())
-		}
-
-		#[cfg(feature = "try-runtime")]
-		fn post_upgrade(_: sp_std::vec::Vec<u8>) -> Result<(), &'static str> {
-			for (asset_id, metadata) in orml_asset_registry::Metadata::<Runtime>::iter() {
-				if matches!(asset_id, CurrencyId::Tranche(_, _)) {
-					frame_support::ensure!(
-						metadata.location.is_none(),
-						"A tranche token's location is not None"
-					)
-				}
-			}
-
-			Ok(())
-		}
-	}
-}
-
-mod evm_chain_id {
-	use pallet_evm_chain_id::ChainId;
-
-	use super::*;
-	use crate::{ParachainInfo, Runtime};
-
-	/// This migration sets the EVM chain ID based on the parachain ID.
-	pub struct SetEvmChainId;
-
-	impl OnRuntimeUpgrade for SetEvmChainId {
-		fn on_runtime_upgrade() -> Weight {
-			let para_id: u32 = ParachainInfo::parachain_id().into();
-			let evm_id: u64 = para_id.into();
-			ChainId::<Runtime>::put(evm_id);
-			<Runtime as frame_system::Config>::DbWeight::get().reads_writes(1, 1)
-		}
-
-		#[cfg(feature = "try-runtime")]
-		fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>, &'static str> {
-			Ok(Default::default())
-		}
-
-		#[cfg(feature = "try-runtime")]
-		fn post_upgrade(_: sp_std::vec::Vec<u8>) -> Result<(), &'static str> {
-			// Migratino is infallible
-			Ok(())
-		}
-	}
-}
+pub type UpgradeAltair1028 = ();
```

Comment:
Just I was thinking about leaving the next migration already named. Really cool!!
Thanks for this PR 🙌🏻